app/org/sagebionetworks/bridge/models/accounts/DataGroups.java

```diff
@@ -0,0 +1,45 @@
+package org.sagebionetworks.bridge.models.accounts;
+
+import java.util.Collections;
+import java.util.Objects;
+import java.util.Set;
+
+import org.sagebionetworks.bridge.models.BridgeEntity;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public final class DataGroups implements BridgeEntity {
+
+    private final Set<String> dataGroups;
+    
+    @JsonCreator
+    public DataGroups(@JsonProperty("dataGroups") Set<String> dataGroups) {
+        this.dataGroups = (dataGroups == null) ? Collections.emptySet() : dataGroups;
+    }
```

Comment:
Do we want to create an immutable copy of dataGroups? This guards against a possible bug where we construct a DataGroups instance, and somewhere else in the code, we modify the underlying set.